### PR TITLE
Feature/vaana 295 initialise client frontend 3

### PR DESCRIPTION
**Pull request by AbdoulayeNDAO121**
Creating pull request for Initialise Client FrontEnd in Repository step 3

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+  <base href="/">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body class="mat-typography">
+  <app-root></app-root>
+</body>
```
Comment (morlombakop) [inline src/index.html:15]: Husky is doing its work 🔥

File: src/app/app.module.ts [hunk 2]
```diff
@@ -2,10 +2,20 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 
 import { AppComponent } from './app.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { AppRoutingModule } from './app-routing.module';
+import {TranslateModule} from '@ngx-translate/core';
```
Comment (morlombakop) [inline src/app/app.module.ts:7]: Use vs-code sort the imports. Absolute import come first

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+
+
```
Comment (morlombakop) [inline src/app/app-routing.module.ts:5]: Should be one line not 3.

Comment (AbdoulayeNDAO121) [reply src/app/app-routing.module.ts:5]: I don't have this in my local branch

Comment (morlombakop) [reply src/app/app-routing.module.ts:5]: @AbdoulayeNDAO121  Then we have a problem,  pull your remote branch it your local or push your local into your remote branch. What goes on the develop is what appear here. It needs to be fixed.

File: src/app/app.module.ts (hunk 2 above)
Comment (morlombakop) [reply src/app/app.module.ts:7]:
```
import { BrowserModule } from '@angular/platform-browser';
import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
import {TranslateModule} from '@ngx-translate/core';

import { AppComponent } from './app.component';
import { AppRoutingModule } from './app-routing.module';
```

File: src/custom-theme.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
+// Custom Theming for Angular Material
```
Comment (morlombakop) [inline src/custom-theme.scss:1]: Let's keep this for the next steps. Please remove it for now.

**Review by morlombakop — APPROVED** [review]

**Pull request merged**